### DISCUSSION TITLE
NordVPN terminal commands

### DISCUSSION THREAD

**Discussion by dinovich** [bubuntux/nordlynx#82] [Q&A]
I'm trying to find a way to make disconnect and reconnect the VPN, to get a new IP address.
`wg-quick  down wg0` won't work for this one obviously.

The `nordvpn` command doesn't seem to be accessible from the terminal. Any way to send a reconnect command? (basically `nordvpn -c`).

Also, is there a way to send the command from a different container?

**Comment by imthenachoman** [comment]
Did you ever figure this out?

**Comment by dinovich** [comment]
Unfortunately I did not.
